templates/panel.json

```diff
@@ -0,0 +1,84 @@
+{
+  "aliasColors": {},
+  "bars": false,
+  "dashLength": 10,
+  "dashes": false,
+  "datasource": "playground",
```

Comment:
It seems that the datasource name should be parameterized.

Reply:
Yes...